bitnami/metrics-server/ci/ct-values.yaml

```diff
@@ -1 +0,0 @@
-values-with-rbac.yaml
```

Comment:
What if you copy the file instead of deleting the symlink?

Reply:
I will try that soon, based on the answer in issue #5800 I assumed it didn't add any value thus could be removed. Going to copy the file and update the PR next week.